diff --git a/panels/panels_mini/panels_mini.module b/panels/panels_mini/panels_mini.module
index d18f097..0c82450 100644
--- a/panels/panels_mini/panels_mini.module
+++ b/panels/panels_mini/panels_mini.module
@@ -7,6 +7,8 @@
  * used within blocks or other panels.
  */
 
+define("PANELS_MINI_BLOCK_SUPPORT", 1);
+
 /**
  * Implementation of hook_permission().
  */
@@ -72,7 +74,7 @@ function panels_mini_form_panels_common_settings_alter(&$form, &$form_state, $fo
     'block_support' => array(
       '#type' => 'checkbox', 
       '#title' => t('Enable Block Support'),
-      '#description' => t('Allows Mini Panles to be exposed as blocks. Please note that this has an impact on performance if enabled.'),
+      '#description' => t('Allows Mini Panels to be exposed as blocks. Please note that this has an impact on performance if enabled.'),
       '#default_value' => $block_support,
     )
   );
@@ -85,9 +87,9 @@ function panels_mini_form_panels_common_settings_alter(&$form, &$form_state, $fo
  * Custom Panels Mini Settings Form Submit Handler
  */
 function panels_mini_settings_form_submit($form, &$form_state) {
-  if(!empty($form_state['values']) && $values = $form_state['values']) {
-    // Store the block support value
-    variable_set('panels_mini_block_support', $values['block_support']);
+  // Store the block support value
+  if(!empty($form_state['values'])) {
+    variable_set('panels_mini_block_support', $form_state['values']['block_support']);
   }
 }
 
@@ -99,8 +101,7 @@ function panels_mini_settings_form_submit($form, &$form_state) {
  */
 function panels_mini_block_info() {
   // Skip if we do not want to support mini panels blocks
-  $panels_mini_block_support = variable_get('panels_mini_block_support', 1);
-  if(!$panels_mini_block_support) return array();
+  if(!variable_get('panels_mini_block_support', PANELS_MINI_BLOCK_SUPPORT)) return array();
 
   // Safety: go away if CTools is not at an appropriate version.
   if (!defined('PANELS_REQUIRED_CTOOLS_API') || !module_invoke('ctools', 'api_version', PANELS_REQUIRED_CTOOLS_API)) {
@@ -180,8 +181,7 @@ function panels_mini_block_configure($delta = 0) {
  */
 function panels_mini_block_list_alter(&$blocks) {
   // Skip if we do not want to support mini panels blocks
-  $panels_mini_block_support = variable_get('panels_mini_block_support', 1);
-  if(!$panels_mini_block_support) return;
+  if(!variable_get('panels_mini_block_support', PANELS_MINI_BLOCK_SUPPORT)) return;
 
   if (module_exists('page_manager')) {
     $current_page = page_manager_get_current_page();
