diff -u --recursive stub/docroot/profiles/crdistro/modules/contrib/feeds_tamper/feeds_tamper.inc /usr/local/zend/var/apps/http/rnd15-local/80/public/sites/all/modules/contrib/feeds_tamper/feeds_tamper.inc
--- stub/docroot/profiles/crdistro/modules/contrib/feeds_tamper/feeds_tamper.inc	2015-05-13 16:34:41.000000000 +0100
+++ /usr/local/zend/var/apps/http/rnd15-local/80/public/sites/all/modules/contrib/feeds_tamper/feeds_tamper.inc	2015-04-07 11:08:52.904749617 +0100
@@ -128,6 +128,10 @@
   // Sort into array, keyed by source.
   $instances = array();
   foreach ($t as $i) {
+    // Add an empty settings array, if the plugin instance doesn't provide any settings itself.
+    if (empty($i->settings)) {
+      $i->settings = array();
+    }
     // Filter disabled and only pull plugins that actually have a corresponding
     // source since we don't delete them with a mapping.
     if (($disabled || empty($i->disabled)) && in_array($i->source, $sources)) {
diff -u --recursive stub/docroot/profiles/crdistro/modules/contrib/feeds_tamper/plugins/absolute_url.inc /usr/local/zend/var/apps/http/rnd15-local/80/public/sites/all/modules/contrib/feeds_tamper/plugins/absolute_url.inc
--- stub/docroot/profiles/crdistro/modules/contrib/feeds_tamper/plugins/absolute_url.inc	2015-05-13 16:34:41.000000000 +0100
+++ /usr/local/zend/var/apps/http/rnd15-local/80/public/sites/all/modules/contrib/feeds_tamper/plugins/absolute_url.inc	2015-04-07 11:08:52.904749617 +0100
@@ -29,7 +29,15 @@
     $dom = new DOMDocument();
   }
 
+  // Supress warnings for invalid HTML.
+  $errors = libxml_use_internal_errors(TRUE);
+  $entity_loader = libxml_disable_entity_loader(TRUE);
+
   $dom->loadHTML($field);
+
+  libxml_use_internal_errors($errors);
+  libxml_disable_entity_loader($entity_loader);
+
   $urls = array();
   $tags = array('a' => 'href', 'img' => 'src', 'iframe' => 'src',
                 'script' => 'src', 'object' => 'codebase', 'link' => 'href',
