diff -rupN placeholder/placeholder.module /usr/local/zend/var/apps/http/rnd15-local/80/public/sites/all/modules/contrib/placeholder/placeholder.module
--- placeholder/placeholder.module	2013-07-01 07:56:41.000000000 +0100
+++ /usr/local/zend/var/apps/http/rnd15-local/80/public/sites/all/modules/contrib/placeholder/placeholder.module	2015-04-07 11:08:53.020749614 +0100
@@ -37,7 +37,7 @@ function placeholder_process_placeholder
     // Add the library, if it's available.
     if (($library = libraries_detect('placeholder')) && !empty($library['installed'])) {
       // Attach the library files.
-      $element['#attached']['libraries_load'][] = array('placeholder', 'minified');
+      $element['#attached']['libraries_load'][] = array('placeholder');
       // Attach the module js file. This will actually invoke the library.
       $element['#attached']['js'] = array(
         drupal_get_path('module', 'placeholder') . '/placeholder.js',
