diff --git a/better_exposed_filters_exposed_form_plugin.inc b/better_exposed_filters_exposed_form_plugin.inc
index 2a1b077..0785189 100644
--- a/better_exposed_filters_exposed_form_plugin.inc
+++ b/better_exposed_filters_exposed_form_plugin.inc
@@ -207,7 +207,7 @@ Title Desc|Z -> A</pre> Leave the replacement value blank to remove an option al
       $bef_specific = FALSE;
 
       // We can work with any of these operators
-      if (in_array($filter->operator, $valid)) {
+      if (in_array($filter->operator, $valid) || ($filter->options['expose']['multiple'] && strpos($filter->table, 'search_api_')===0)) {
         $bef_specific = TRUE;
       }
       // ... or any of these specific filters
