diff -uprb a/plugins/context_block_disable_reaction.inc b/plugins/context_block_disable_reaction.inc
--- plugins/context_block_disable_reaction.inc	2013-01-16 00:41:55.000000000 +0000
+++ plugins/context_block_disable_reaction.inc	2015-04-07 11:08:52.844749618 +0100
@@ -79,7 +79,7 @@ class Context_Block_Disable_Reaction ext
    *
    * @see hook_block_view_alter()
    */
-  public function execute(&$data, $block) {
+  public function execute(&$data, $block = NULL) {

     $contexts = $this->get_contexts();


