--- a/eu_cookie_compliance.admin.inc	
+++ b/eu_cookie_compliance.admin.inc	
@@ -27,6 +27,12 @@ function eu_cookie_compliance_admin_form
     '#default_value' => isset($popup_settings['popup_enabled']) ? $popup_settings['popup_enabled'] : 0,
   );

+  $form['eu_cookie_compliance_' . $ln]['post_enabled'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Enable for POST requests'),
+    '#default_value' => isset($popup_settings['post_enabled']) ? $popup_settings['post_enabled'] : 1,
+  );
+
   $form['eu_cookie_compliance_' . $ln]['popup_clicking_confirmation'] = array(
     '#type' => 'checkbox',
     '#title' => t('Consent by clicking'),

--- a/eu_cookie_compliance.module	
+++ b/eu_cookie_compliance.module	
@@ -36,7 +36,13 @@ function eu_cookie_compliance_init() {
     $geoip_match = true;
   }

-  if (!empty($popup_settings['popup_enabled']) && user_access('display EU Cookie Compliance popup') && $geoip_match) {
+  if (empty($popup_settings['post_enabled'])) {
+    $processposts = ('POST' != $_SERVER['REQUEST_METHOD']);
+  } else {
+    $processposts = true;
+  }
+
+  if (!empty($popup_settings['popup_enabled']) && user_access('display EU Cookie Compliance popup') && $geoip_match && $processposts) {
     global $language;

     // color overrides
